docker-compose.yml

```
@@ -0,0 +1,20 @@
+---
+version: "3.9"
+
+services:
+
+  materialized:
+    image: materialize/materialized:latest
+    container_name: materialized
+    ports:
+      - 6875:6875
+    healthcheck: {test: curl -f localhost:6875, interval: 1s, start_period: 30s}
```

I believe that the problem is with the healthcheck.
Basically it takes ~10s or so for the materialized container to be fully ready, but only using the `depends_on` directive, would wait for just a second or so.



```suggestion
      - 6878:6878
    healthcheck: {test: curl -f localhost:6878/api/readyz, interval: 1s, start_period: 30s}
```

---

I've managed to reproduce the failure locally, after running the test a few times in a row until I hit the same error. My guess is that locally the materialized container becomes healthy quicker so we don't hit the problem as often.